src/Game.java

```diff
@@ -0,0 +1,160 @@
+import jodd.json.JsonParser;
+import jodd.json.JsonSerializer;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * Created by Nigel on 6/21/16.
+ */
+public class Game {
+    File file = new File("players.json");
+    private int totalSticks;
+    private Player player1, player2, currentPlayer = null;
+    Scanner scanner = new Scanner(System.in);
+    public static final int STICKMAX = 100;
+    public static final int STICKMIN = 10;
+    public static final int SELECTMIN = 1;
+    public static final int SELECTMAX = 3;
+    String winnerName;
```

Comment:
The name being saved seems less than efficient.  The only to get the player object from this is to loop through all the players and check the name.  It would probably be better served to store the `winner` and `loser` as references to the different player objects.